packages/data-point-service/README.md

```diff
@@ -19,74 +19,72 @@ $ npm install data-point-cache
 
 Create a new DataPoint Service Object
 
-
```

Comment:
truly sorry for all of these changes, it seems the file had not been ran via prettier before. my only change on this file is marked at the bottom of the file